spec/system/signup_spec.rb

```diff
@@ -96,7 +96,7 @@
           .fill_code("pudding")
         expect(signup_form).to have_valid_fields
 
-        signup_form.click_create_account
+        signup_form.click_create_account(expect_success: false)
```

Comment:
This seems like an oversight from before, which strangely didn't manifest until now. When using an incorrect invite code, we shouldn't expect success.